tools/packaging/packaging.psm1

```diff
@@ -1426,7 +1426,8 @@ function New-UnifiedNugetPackage
         "Microsoft.PowerShell.SDK.dll",
         "Microsoft.WSMan.Management.dll",
         "Microsoft.WSMan.Runtime.dll",
-        "System.Management.Automation.dll")
+        "System.Management.Automation.dll",
```

Comment:
NIT:  with this format, the commas aren't needed and if you close it on a new line you won't delete a line each time you add an entry.

Reply:
👍 Don't what to make this change now if you are ok with it.